pkg/component/runtime/command.go

```diff
@@ -20,8 +25,14 @@ import (
 type actionMode int
 
 const (
-	actionStart = actionMode(0)
-	actionStop  = actionMode(1)
+	actionStart    = actionMode(0)
```

Comment:
default uninitialized value for actionMode is start, not sure if this is ok but i don't like it tbh

Reply:
Changed it to be 0 for stop, -1 for teardown, and 1 for start.